packages/story-editor/src/components/button/index.js

```diff
@@ -17,13 +17,13 @@
 /**
  * External dependencies
  */
-import { forwardRef } from 'react';
+import React, { forwardRef } from 'react';
 import styled from 'styled-components';
 
 /**
  * Internal dependencies
  */
-import { ArrowDown as ArrowDownIcon, More as MoreIcon } from '../../icons';
+import { Icons } from '../../../../design-system';
```

Comment:
Design system components should be imported via the `@web-stories-wp/design-system` alias